requirements/dev.in

```diff
@@ -2,3 +2,4 @@
 
 tox
 tox-battery
+black
```

Comment:
Since black doesn't work under Python 3.5, this will force dev environments to use Python 3.6+ .  That's a little less than ideal if we want to keep running `make upgrade` under 3.5 since that's our primary target version for now.